stdlib/doc/sphinx/language/coq-library.rst

```diff
@@ -0,0 +1,393 @@
+The standard library
+--------------------
+
+Survey
+~~~~~~
+
+The rest of the standard library is structured into the following
```

Comment:
It seems a bit weird to start a page of doc by "The rest of..." (without referencing the prelude first).